commit ed5d1ea7e36575a06d134f06b52abd6e8d7c4e35 (refs/changes/29/5029/4)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-12-21T01:14:27+00:00 (10 months ago)
    
    MANTA-3956 add muskie_delete_bytes metric to Muskie docs
    Reviewed by: Robert Bogart <robert.bogart@joyent.com>
    Approved by: Robert Bogart <robert.bogart@joyent.com>

diff --git a/docs/internal/design.md b/docs/internal/design.md
index ebd3d6c..1af078a 100644
--- a/docs/internal/design.md
+++ b/docs/internal/design.md
@@ -69,6 +69,7 @@ The following metrics are collected:
 - End-to-end latency for all requests
 - Count of requests completed
 - Count of bytes streamed to and from storage
+- Count of object bytes deleted
 
 Each of the metrics returned include the following metadata labels:
 
